src/functions/usdarrayconcat.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Merge two or more arrays. 
+---
+
+# $arrayConcat
+
+`$arrayConcat` used to merge two or more arrays without does not changing the existing arrays, but instead returning a new array.
+
+### Usage 
+
+```php
+$arrayConcat[separator;arrayName...]
+```
+
+### Properties
+
+| FIELD | TYPE | DESCRIPTION |
+| ----- | ----- | ----- | 
+| separator | string | The separator for array elements | 
+| arrayName... | string | The arrays going to be merged with separated semicolon |
+
+## Example
+
+```javascript
+bot.command({
+  name: "array-concat", 
+  code: `
+  $createArray[newArray;$arrayConcat[,;array1;array2]]
```

Comment:
wrong example.

just change this to 

`concated result: $arrayConcat[,;array1;array2]`